liblwgeom/lwcollection.c

```diff
@@ -494,9 +495,9 @@ int lwcollection_is_empty(const LWCOLLECTION *col)
 }
 
 
-int lwcollection_count_vertices(LWCOLLECTION *col)
+uint32_t lwcollection_count_vertices(LWCOLLECTION *col)
 {
-	int i = 0;
+	uint32_t i = 0;
 	int v = 0; /* vertices */
```

Comment:
Yes. I've changed v to uint32_t and added a TODO comment in `lwgeom_count_vertices` since both the internal functions `lwpoly_count_vertices` and `lwcollection_count_vertices`. Should be fixed by using uint64_t, I think.